### ISSUE TITLE
Cannot play back any music in directory view

### ISSUE THREAD

**Issue by zDEFz** [jellyfin/jellyfin-web#4295]
**Media Info of the file**

NO file involved

**Logs**

https://termbin.com/cmr3

**FFmpeg Logs**

No file involved

**Stats for Nerds Screenshots**

No file involved

**Server System (please complete the following information):**
 - OS: [Docker on AsahiLinux]
 - Jellyfin Version: [e.g. 10.8.1]
 - Hardware settings & device: [Mac Mini m1 on ASAHILINUX ]
 - Reverse proxy: [none]
 - Other hardware notes: [none]
 
**Client System (please complete the following information):**
 - Device: [custom AMD gaming PC]

https://user-images.githubusercontent.com/24463722/183839048-35c4d157-63d2-4756-bc3f-579479ea7995.mp4

 - OS: [Windows 10]
 - Client: [Firefox 103.0.1]
 - Browser (if Web client): [Firefox]
 - Client and Browser Version: [10.8.1 and 103.0.1]

**Comment by zDEFz** [comment]
any news?

**Comment by thornbill** [comment]
I don't think there have been any changes to this. Can you describe what you are experiencing? The videos seem to be doing several different things so perhaps there are multiple issues here? Thanks!

**Comment by zDEFz** [comment]
> I don't think there have been any changes to this. Can you describe what you are experiencing? The videos seem to be doing several different things so perhaps there are multiple issues here? Thanks!

The issue is that Jellyfin requires a certain structure, including the tags of the file itself to be to it's liking.
Only if I go into the file view, that does no longer matter and I can play back file by file.
Jellyfin is too opinionated. Especially for Music that wouldn't be on a public music tag archive.

It is overall - similar to Kodi, but with a key difference!
In Kodi, you can name your files within the TheTVDB format, and it will be properly recognized, most of the time.

If I specify folder-view in Jellyfin, I expect the files to be shown as they are, and when I try to play back a file, it should resolve into the absolute and correct filepath. Not something that is relying on tags.
The video above does illustrate this well.

Moreover, Jellyfin should have full shift_jis support [...]

**Comment by zDEFz** [comment]
> So you want the folder view to essentially function like a file browser?

I'd like the `folder-view` to display as they are on disk.
If we have the option to display like a file browser would, that would be okay, as well.

But in any case, there should be a clear indication to the user if that is not the case.
Because then, we would get reports of a file not being playable, because the path is incorrect.

As a user, when I 'scan' my library, I do expect Jellyfin to add everything that can be played back, and when I enable folder-view, I'd like to see all files that can be played back, within the folders of my structure.

**Comment by thornbill** [comment]
Sounds like [this feature request](https://features.jellyfin.org/posts/224/add-a-folder-view-to-libraries).

**Comment by RobinEd** [comment]
Yes, I would like to see my files and folders - I have been curating my music collection for years in a folder structure.

**Comment by thornbill** [comment]
Closing since we don’t track feature requests in GitHub issues.
